apps/mobile/src/features/AccountsSheet/AccountItem/AccountItem.tsx

```diff
@@ -52,7 +52,23 @@ export function AccountItem({ account, drag, chains, isDragging, activeAccount,
         <AccountCard
           leftNode={
             isEdit && (
-              <TouchableOpacity onPress={onSafeDeleted(account.address.value as Address)}>
+              <TouchableOpacity
+                onPress={() => {
+                  Alert.alert('Delete Safe', 'Are you sure you want to delete this safe?', [
```

Comment:
Can we move this function to be outside the render layer?

Reply:
good point. I guess I was in a hurry yesterday evening :)